frontend/src/routes/FacilityBooking/CreateBooking/index.tsx

```diff
@@ -220,6 +220,18 @@ export default function CreateBooking() {
 
   locationOptions.push({ value: 'Conference Room' })
 
+  const {
+    register,
+    handleSubmit,
+    getValues,
+    formState: { errors },
+  } = useForm()
+
+  const submitForm = (data) => {
+    console.log(getValues())
```

Comment:
if ure just looking to see values in the input fields, 'watch()' from react hook form works too - can put in use effect to trigger

```
useEffect(() => {
    console.log(watch())
}, [watch])
```